pal_event_server/lib/src/loggers/app_usage/macos/apple_script_util.dart

```diff
@@ -39,13 +39,13 @@ const scriptArgs = [
   "-e", r'global frontApp, frontAppName, windowTitle', //
   "-e", r'set windowTitle to ""',
   "-e", r'tell application "System Events"',
-  "-e", r'set frontApp to first application process whose frontmost is true',
-  "-e", r'set frontAppName to name of frontApp',
+  "-e", r'set frontAppId to (id of first application process whose frontmost is true)',
+  "-e", r'set frontAppName to title of application process id frontAppId',
   "-e",
-  r'set axwindows to (every UI element of frontApp whose role is "AXWindow" or subrole is "AXStandardWindow")',
+  r'set axwindows to (every UI element of application process id frontAppId whose role is "AXWindow" or subrole is "AXStandardWindow")',
   "-e", r'if axwindows is not {} then',
   "-e", r'set windowTitle to name of (item 1 of axwindows)',
-  "-e", r'if frontAppName is "Google Chrome" then',
+  "-e", r'if frontAppName is "Chrome" then',
   "-e", r'tell application "Google Chrome"',
```

Comment:
Will this work if it matches frontAppName to "Chrome"?

Reply:
Yes, it works.